app/views/site/how_it_works_newbie.erb

```diff
@@ -106,18 +106,11 @@
   <div class="row">
     <div class="col-md-8">
       <p>
-      This might be the trickiest part of Exercism.
-      </p>
-
-      <p>
       How you install the Exercism CLI depends on what kind of computer you have.
-      We've explained all of this in detail on the <a href="/cli">CLI page</a>, but
-      it's pretty confusing.
+      We've explained all of this in detail on the installation page (link, below).
```

Comment:
Your second point hadn't occurred to me, @jtigger, but I think you're probably right.

Can we have both? Or is that more confusing.